lib/plugins/filter/after_post_render/external_link.js

```diff
@@ -16,21 +16,17 @@ function externalLinkFilter(data) {
   if (config.external_link === false || config.external_link.enable === false
     || config.external_link.field !== 'post') return;
 
-  const exclude = Array.isArray(config.external_link.exclude) ? config.external_link.exclude
-    : [config.external_link.exclude];
-
-  data.content = data.content.replace(/<a .*?(href=['"](.+?)['"]).*?>/gi, (str, hrefStr, href) => {
-    if (/target=/gi.test(str) || !isExternalLink(href, config.url, exclude)) return str;
+  data.content = data.content.replace(/<a([\s]+|[\s]+[^<>]+[\s]+)href=["']([^<>"']+)["'][^<>]*>/gi, (str, _, href) => {
```

Comment:
Optimize as in only limit to certain character, instead of `.*` any character?

Reply:
@curbengh 

Yeah. It seems that using `([\s]+|[\s]+[^<>]+[\s]+)` (instead of ` .*`) is a bit faster.